reset.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rm -rf ~/.sbt/boot/scala-2.11.8/org.scala-sbt/sbt/1.0.0-SNAPSHOT/
```

Comment:
That was on purpose. We can put it in `scripts` directory or something to avoid clutter.